aws-iot-mitigationaction/aws-iot-mitigationaction.json

```diff
@@ -0,0 +1,253 @@
+{
+  "typeName": "AWS::IoT::MitigationAction",
+  "description": "Mitigation actions can be used to take actions to mitigate issues that were found during an audit.",
```

Comment:
Mitigation Action isn't specific to Audit, now even ML Detect uses it. So lets paraphrase

Reply:
Changed